### PR TITLE
Fix VideoSinkApple (AVFoundation synchronization)

### PR DESCRIPTION
**Pull request by headupinclouds**
We need to “capture” a local copy of the handler input variable and not the reference.  Avoid drawing the lower face features until the compact models are more accurate.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,7 @@ set(drishti_upload_init "${CMAKE_CURRENT_LIST_DIR}/drishti-upload/init.cmake")
 if(DRISHTI_USE_DRISHTI_UPLOAD)
   if(EXISTS "${drishti_upload_init}")
     # CI: Locked CI build configuration
+    include("cmake/HunterGate.cmake") # HUNTER_ENABLED
```
Comment (ruslo) [inline CMakeLists.txt]: Not needed, path to HunterGate will be added here:
* https://github.com/elucideye/drishti-upload/blob/3ee5b46cdd63c86cbbd8c534e5955fec9afb72f0/init.cmake#L36

Comment (headupinclouds) [reply CMakeLists.txt]: Something still needs to include it though.  If I remove that line and test with `DRISHTI_USE_DRISHTI_UPLOAD=YES` I get the following error:

```
HUNTER_RUN_UPLOAD: OFF
CMake Error at drishti-upload/init.cmake:38 (HunterGate):
  Unknown CMake command "HunterGate".
Call Stack (most recent call first):
  CMakeLists.txt:23 (include)
```

AFAIK, adding `cmake/Modules` to `CMAKE_MODULE_PATH` just allows us to shorten `include("cmake/HunterGate.cmake")` to `include("HunterGate.cmake")` after the `CMAKE_MODULE_PATH` has been updated.

Comment (headupinclouds) [reply CMakeLists.txt]: Going to merge for the fix w/ default `DRISHTI_USE_DRISHTI_UPLOAD`.  Let me know if you see a follow-up improvement.

Comment (ruslo) [reply CMakeLists.txt]: My bad, we need this include indeed.

**Pull request merged**